### PR TITLE
fix: add pod and workflow workers to argo helm values

### PR DESCRIPTION
**Pull request by AntoineDao**
Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo/values.yaml [hunk 1]
```diff
@@ -57,6 +57,9 @@ controller:
   #  spec:
   #    ttlStrategy:
   #      secondsAfterCompletion: 84600
+  scaling: {}
```
Comment (stefansedich) [inline charts/argo/values.yaml:60]: I am not sure the `scaling` secion is 100% nescassary and keeping it directly under `controller` would suffice.

File: charts/argo/templates/workflow-controller-deployment.yaml [hunk 2]
```diff
@@ -43,6 +43,14 @@ spec:
           {{- if .Values.singleNamespace }}
           - "--namespaced"
           {{- end }}
+          {{- if .Values.controller.scaling.workflowWorkers }}
```
Comment (stefansedich) [inline charts/argo/templates/workflow-controller-deployment.yaml:46]: You can use {{ with }} here to simplify things:

```
{{- with .Values.controller.podWorkers }}
- "--pod-workers"
- {{ . | quote }}
{{- end }}
```

File: charts/argo/values.yaml (hunk 1 above)
Comment (AntoineDao) [reply charts/argo/values.yaml:60]: Yep makes sense. Was thinking of making it "thematic" but it probably makes for unnecessary nesting.

File: charts/argo/templates/workflow-controller-deployment.yaml (hunk 2 above)
Comment (AntoineDao) [reply charts/argo/templates/workflow-controller-deployment.yaml:46]: Yep definitely cleaner. Will implement.

**Pull request merged**